pkg/pillar/types/zedmanagertypes.go

```diff
@@ -69,6 +69,8 @@ type AppInstanceConfig struct {
 	// XXX: to be deprecated, use CipherBlockStatus instead
 	CloudInitUserData *string // base64-encoded
 	RemoteConsole     bool
+	// Collect Stats IP Address
+	CollectStatsIPAddr string
```

Comment:
Would it be easier to track this as a net.IP instead of string?
From the string definition it isn't clear to me whether it should/does include a TCP port number etc aka net.TCPAddr